.g8/textInputPage/app/pages/$className$Page.scala

```diff
@@ -0,0 +1,25 @@
+package pages
+
+import play.api.libs.json.JsPath
+import models.SchemeId.Srn
+import models.Money
+$if(!index.empty)$
+import config.Refined.$index$
+import utils.RefinedUtils._
+import eu.timepit.refined.refineMV
+$endif$
+
+$if(index.empty)$
+case class $className;format="cap"$Page(srn: Srn) extends QuestionPage[String] {
```

Comment:
Are these changes intentional @narendrapaduchuri ?

Reply:
I think this is system generated ones, I haven't added that